### PR TITLE
Update libs, code, license

### PR DESCRIPTION
**Pull request by koji-1009**
- Kotlin 1.4.30
- OkHttp 4.9.1
- Latest AndroidX libs

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -15,7 +15,5 @@ org.gradle.jvmargs=-Xmx1536m
 # Android operating system, and which are packaged with your app's APK
 # https://developer.android.com/topic/libraries/support-library/androidx-rn
 android.useAndroidX=true
-# Automatically convert third-party libraries to use AndroidX
-android.enableJetifier=true
```
Comment (nacatl) [inline gradle.properties:19]: 👏

File: sdk-example-java/.gitignore [hunk 2]
```diff
@@ -1 +0,0 @@
-/build
```
Comment (nacatl) [inline sdk-example-java/.gitignore:1]: R.I.P.

**Review by nacatl — APPROVED** [review]
LGTM

**Review by JASON13F — APPROVED** [review]
LGTM

**Pull request merged**